store/views.py

```diff
@@ -61,10 +73,18 @@ def loanBookView(request):
     Check if an instance of the asked book is available.
     If yes, then set the message to 'success', otherwise 'failure'
     '''
-    # START YOUR CODE HERE
-    book_id = None # get the book id from post data
-
-
+    book_id =request.POST['bid']# get the book id from post data
```

Comment:
You directly access POST data without checking if it even exists. This may lead to server crash if a user access this endpoint with invalid request data. The good behavior would have been to throw a client error (400), rather than server error (500).